packages/fxa-settings/src/models/integrations/web-integration.ts

```diff
@@ -26,7 +26,10 @@ export class BaseIntegrationData extends ModelDataProvider {
   context: string | undefined;
 
   @IsOptional()
-  @IsEmail()
+  // TODO: change this to 'IsEmail'. Requiring this to be an email causes
```

Comment:
Can this start with `// TODO - Validation`. Probably search for those when revisiting this.